thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/mapper/AdditionalInfoMapper.java

```diff
@@ -0,0 +1,14 @@
+package pl.edu.agh.quizzesthesis.business.mapper;
+
+import org.springframework.stereotype.Component;
+import pl.edu.agh.quizzesthesis.api.dto.AdditionalInfoResponse;
+import pl.edu.agh.quizzesthesis.data.entity.Term;
+
+@Component
+public class AdditionalInfoMapper {
```

Comment:
Again, `AdditionalInfo` can be everything and we can't have a mapper that covers only one case.